lnbits/extensions/boltz/mempool.py

```diff
@@ -0,0 +1,97 @@
+import asyncio
```

Comment:
This file is in the Boltz extension. Wouldn't it be useful other extensions too if we make a little more generic?

Reply:
i am planning on doing a mempool extension, which will take care of things like that.

Reply:
there is this proposal of mine i did in april: https://github.com/lnbits/lnbits-legend/issues/584